src/components/ExpandableMeetings.js

```diff
@@ -74,37 +75,41 @@ export default function ExpandableMeetings(props) {
     const meeting = meetings[0]
     return (
       <div key={meeting.endDate + Math.random()} className={classes.meetBorder}>
-        {displayLink(meeting.buildingRoom, meeting.campus) && (
+        {displayLink(meeting.location, meeting.campus) && (
           <a
             className={classes.meetLink}
-            tabIndex="0"
-            target="_blank"
-            href={getMapUrl(meeting.buildingRoom, false)}
-            rel="noopener noreferrer"
+            tabIndex='0'
+            target='_blank'
+            href={getMapUrl(meeting.location, false)}
```

Comment:
Ooh. Also something related to this repo but not this exact PR. It might be a good task for someone to go through and update the mapUrls - especially if the new dorm is on there now

Reply:
True that